ros_ws/src/simulation/racer_world/launch/rviz_config.rviz

```diff
@@ -4,18 +4,16 @@ Panels:
     Name: Displays
     Property Tree Widget:
       Expanded:
-        - /Status1
```

Comment:
All the following is just rviz-config stuff like in the other file - same comments